pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta:__legacy__"
+
+[tool.isort]
+profile = "black"
+
```

Comment:
do you need to set this? I think it only affects direct uses of isort, not ruff's reimplementation which seems to just do the `black` profile settings by default (see https://github.com/astral-sh/ruff/issues/465)